interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
+ //Get new_validate option to check if global new validation is on.
+ $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
+ //Gets validation rules from Page Validation list.
+ $collectthis = collectValidationPageRules("/interface/forms/newpatient/new.php");
+ ?>
+ <?php if($new_validate && $collectthis): ?>
```

Comment:
Ok, I took out the new validation check - now it depends only on the 'active' trigger in the list itself.